components/games/freeciv/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Jake Goerzen
+#
+
+BUILD_BITS= 64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		freeciv
+COMPONENT_VERSION=	2.6.5
+COMPONENT_SUMMARY=	Free and Open Source empire-building strategy game inspired by the history of human civilization.
+COMPONENT_PROJECT_URL=	https://www.freeciv.org
```

Comment:
Minor note, https seems to be not working on the site.

Reply:
I've changed COMPONENT_PROJECT_URL from https to http, so all should be in order now.